docs/source/etl/variant-splitter.rst

```diff
@@ -70,5 +70,10 @@ Assuming ``df_original`` is a variable of type DataFrame which contains the geno
 
             df_split = Glow.transform("split_multiallelics", df_original)
 
+.. tip::
+
+    The ``split_multiallelics`` transformer is often significantly faster if the `whole-stage code generation` feature of Spark Sql is turned off. Therefore, it is recommended that you turn off this feature using the command ``spark.conf.set("spark.sql.codegen.wholeStage", False)`` before using this transformer.
```

Comment:
This is language-specific, right? If so we should have a Python and a Scala version.

Reply:
tabs created.

Reply:
Looks good!

Reply:
Maybe it would be good to remind the user to turn codegen back on after as well.

Reply:
Maybe just say `that you temporarily turn off`

Reply:
Added remember warning